documentation/book/assembly-security.adoc

```diff
@@ -0,0 +1,38 @@
+// This assembly is included in the following assemblies:
+//
+// master.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+:parent-context: {ContextProduct}
+
+include::common/attributes.adoc[]
+
+[id='security-{context}']
+= Security
```

Comment:
Right now, this calls only about encryption. Do we plan add the other modules about Authentication and Authorization here as well in the future? If not, we should probably rename it to just _Encryption_ instead of _Security_.